DemoApp/appcenter-post-clone.sh

```diff
@@ -2,3 +2,6 @@
 echo "Executing post clone script in `pwd`"
 echo $NPM_RC | base64 --decode > $APPCENTER_SOURCE_DIRECTORY/DemoApp/.npmrc
 echo $GOOGLE_SERVICES_JSON | base64 --decode > $APPCENTER_SOURCE_DIRECTORY/DemoApp/android/app/google-services.json
```

Comment:
Did you migrate demo to remove Firebase like in TestApp?

Reply:
DemoApp is migrated, thus GOOGLE_SERVICES_JSON is no longer needed.